cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/setup/CheckImageRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.cloud.event.setup;
+
+import com.sequenceiq.cloudbreak.cloud.event.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.event.resource.CloudStackRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+
+public class CheckImageRequest<T> extends CloudStackRequest<T> {
```

Comment:
extends CloudStackRequest<CheckImageResult> instead of  extends CloudStackRequest<T>